### ISSUE TITLE
[Bug] stopMomentum on Flight unregisters keypresses

### ISSUE THREAD

**Issue by Cypphi** [AntiCope/meteor-rejects#64]
### Describe the bug

Toggling off Flight while holding a key will unregister the key so you have to press the key again to be able to walk.

I looked at the code and tried fixing it but can't really find a way, I understand why this happens but can't think of a way myself to fix it.

### Steps to reproduce

```plaintext
1. enable stopMomentum in the flight module
2. toggle on flight
3. fly
4. toggle off flight (while pressing W,A,S,D)
5. You're not moving in the direction of what key you're pressing
```

### Operating System

Windows

### Rejects Version

latest

### Meteor Version

latest dev build

### Minecraft Version

1.17.1

### Other mods

```plaintext
none
```

**Comment by StormyBytes** [comment]
Probably due to [this](https://github.com/AntiCope/meteor-rejects/blob/master/src/main/java/cloudburst/rejects/mixin/meteor/modules/FlightMixin.java#L41-L44)

A fix I can think off is to cancel player velocity (set to 0) instead of setting the keypress to false

**Comment by C10udburst** [comment]
it is setting player velocity to 0 right [here](https://github.com/AntiCope/meteor-rejects/blob/1973365691f0b641514500d883604677775cdd05/src/main/java/cloudburst/rejects/mixin/meteor/modules/FlightMixin.java#L46), but it doesn't work properly without unregistering keypresses

**Comment by Cypphi** [comment]
> it is setting player velocity to 0 right [here](https://github.com/AntiCope/meteor-rejects/blob/1973365691f0b641514500d883604677775cdd05/src/main/java/cloudburst/rejects/mixin/meteor/modules/FlightMixin.java#L46), but it doesn't work properly without unregistering keypresses

Yeah what's the deal with that? I actually tried fixing it myself and just setting velocity to 0 but it doesn't work for some weird reason...
